src/utils.ts

```diff
@@ -12,24 +12,21 @@ import {
   PopupCancelledError
 } from './errors';
 
-export const parseQueryResult = (queryString: string): AuthenticationResult => {
+export const parseAuthenticationResult = (
```

Comment:
Renamed this to `parseAuthenticationResult`, as it's not a generic querystring parser.